### PR TITLE
Formatting diff editor title

### PR DESCRIPTION
**Pull request by kisantia**
Add background color and other formatting to diff editor title to match look of OE. Also changing title for schema compare diff editor to "Compare Details" instead of "Object Definitions"

light theme:
image

dark theme:
image

high contrast black:
image

### CODE REVIEW

File: src/sql/workbench/electron-browser/modelComponents/media/editor.css [hunk 1]
```diff
@@ -15,10 +15,19 @@ modelview-diff-editor-component {
...
+.modelview-diff-editor-title {
+	width: 100%;
+	height: 100%;
+	font-weight: 700;
+	padding: 3px 3px 3px 16px;
+	text-transform: uppercase;
+	font-size: 11px
+}
+
+.vs-dark .modelview-diff-editor-title-background {
```
Comment (alanrenmsft) [inline src/sql/workbench/electron-browser/modelComponents/media/editor.css]: please handle .hc-black as well.

Comment (kisantia) [reply src/sql/workbench/electron-browser/modelComponents/media/editor.css]: added

**Comment by udeeshagautam** [comment]
@kisantia of the two dark theme pics.. which one is after change? this one right?
image

**Comment by kisantia** [comment]
@udeeshagautam the second one is dark theme and the third one is high contrast black. I updated the comment to say what each of them are.

**Pull request merged**